analysis/warrior/src/Condemn.ts

```diff
@@ -8,10 +7,12 @@ class Condemn extends Analyzer {
     abilities: Abilities,
   };
 
+  condemnSpellId: number = 0;
+
   constructor(options: Options) {
     super(options);
 
-    this.active = this.selectedCombatant.hasCovenant(COVENANTS.VENTHYR.id);
+    this.active = false; // this.selectedCombatant.hasCovenant(COVENANTS.VENTHYR.id);
```

Comment:
if this module is just going to be disabled, it should simply be removed.

Reply:
Ah right. D'oh! Removed.